python/ccxt/bithumb.py

```diff
@@ -617,7 +617,7 @@ def fetch_trades(self, symbol: str, since: Optional[int] = None, limit: Optional
         self.load_markets()
         market = self.market(symbol)
         request = {
-            'currency': market['base'],
+            'currency': f'{market["base"]}_{market["quote"]}',
```

Comment:
Hello @choisangh, next time pls check the contribution guides we need to apply the fix to the TS version in a compatible way.